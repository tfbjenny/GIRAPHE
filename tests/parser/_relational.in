true and true;
true or false;
