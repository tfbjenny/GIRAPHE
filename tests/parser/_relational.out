Expr(Binop(Num_Lit(1), EQ, Num_Lit(1)));
Expr(Binop(Num_Lit(1), NEQ, Num_Lit(1)));
Expr(Binop(Num_Lit(1), LT, Num_Lit(1)));
Expr(Binop(Num_Lit(1), LET, Num_Lit(1)));
Expr(Binop(Num_Lit(1), GT, Num_Lit(1)));
Expr(Binop(Num_Lit(1), GET, Num_Lit(1)));
Expr(Binop(Bool_lit(true), AND, Bool_lit(true)));
Expr(Binop(Bool_lit(true), OR, Bool_lit(false)));
